src/svg.erl

```diff
@@ -258,6 +258,16 @@ path([{X0, Y0} | Tail], Attr) ->
    Path  = [ [$M, 32, coord(X0), 32, coord(Y0)] | Ptail ],
    {path, [{d, lists:flatten(Path)} | Attr], []}.
 
+%%
+%% line(Point, Point) -> Element
+%%
+%% defines an ellipse based on center point and two radii. 
```

Comment:
I think this comment has been copied from `ellipse` function

Reply:
Sorry about the sloppy pr, I updated it now.